packages/eslint-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spotify/eslint-plugin",
-  "version": "8.1.0",
+  "version": "8.1.1",
```

Comment:
@Keysox you don't actually need to (and shouldn't) set a new version - the semantic-release version determines whether or not a release occurs in CI.

I'm going to replace that for you. If you want this to trigger a patch release, the semantic-release commit type should be `fix`. [Read more here](https://www.conventionalcommits.org/en/v1.0.0/).